tiptapy/__init__.py

```diff
@@ -104,6 +107,29 @@ class Doc(BaseContainer):
     type = "doc"
 
 
+class OrderedList(ListItem, Paragraph):
+    type = "ordered_list"
+    wrap_tag = "ol"
+
+    def inner_render(self, node):
+        list_items, paragraph_items = ListItem(), Paragraph()
+        list_contents = [
+            text['text']
+            for list_item in node['content']
+            for text in list_item['content'][0]['content']
+        ]
+        html = ""
+        for content in list_contents:
+            html += """<{}><{}>{}</{}></{}>""".format(
```

Comment:
@Pradhvan it is not necessary to always get a `p` tag inside `li`, we can get any tag.